templates/apps/actor-type.hbs

```diff
@@ -4,6 +4,14 @@
     <div class="form-group stacked">
         <label>{{ localize "DND5E.CreatureType" }}</label>
         <ul class="trait-list">
+            {{#if automatic}}
+              <li>
+                <label class="radio">
+                  <input type="radio" name="value" value="" {{checked type.chosen}}>
+                  {{automatic}}
+                </label>
+              </li>
+            {{/if}}
```

Comment:
Seems this is not used? I can't find a reference to `automatic` in `ActorTypeConfig` anywhere.

Reply:
Whoops, leftover from another approach I was attempting.